app/src/main/java/com/android/voyageur/ui/overview/Overview.kt

```diff
@@ -135,119 +142,142 @@ fun TripItem(tripsViewModel: TripsViewModel, trip: Trip, navigationActions: Navi
+                    contentDescription = "Selected image",
+                    contentScale = ContentScale.Crop,
+                    modifier = Modifier.width(120.dp).height(217.dp).testTag("tripImage"))
+              } else {
+                Image(
+                    painter = painterResource(id = R.drawable.default_trip_image),
+                    contentDescription = "Trip image overview",
+                    contentScale = ContentScale.Crop,
+                    modifier = Modifier.width(120.dp).height(217.dp).testTag("defaultTripImage"))
+              } // modifier.weight(2f) is used here to set the column to 2/3 of the card
               Column(
                   modifier = Modifier.fillMaxSize().padding(16.dp).weight(2f),
                   verticalArrangement = Arrangement.Top) {
                     Text(
                         modifier = Modifier.fillMaxWidth(),
                         text = trip.name,
                         textAlign = TextAlign.Start,
-                        style = MaterialTheme.typography.titleLarge,
-                        color = MaterialTheme.colorScheme.onSurfaceVariant)
-
+                        style =
+                            TextStyle(
+                                fontSize = 23.sp,
+                                lineHeight = 20.sp,
+                                fontWeight = FontWeight(500),
+                                color = themeColor,
+                                letterSpacing = 0.23.sp))
                     Text(
-                        modifier = Modifier.fillMaxWidth().padding(top = 4.dp),
+                        modifier = Modifier.fillMaxWidth().padding(start = 15.dp, top = 4.dp),
                         text = dateRange,
                         textAlign = TextAlign.Start,
-                        style = MaterialTheme.typography.bodySmall,
-                        color = MaterialTheme.colorScheme.onSurfaceVariant.copy(alpha = 0.7f))
-
-                    DisplayParticipants(trip)
+                        style =
+                            TextStyle(
+                                fontSize = 10.sp,
+                                lineHeight = 20.sp,
+                                fontWeight = FontWeight(500),
+                                color = themeColor,
+                                letterSpacing = 0.1.sp,
+                            ))
+                    DisplayParticipants(trip, userViewModel)
                   }
             }
-      }
+      })
 }
 
 @SuppressLint("StateFlowValueCalledInComposition")
 @Composable
-fun DisplayParticipants(trip: Trip) {
-  val configuration = LocalConfiguration.current
-  val screenWidth = configuration.screenWidthDp.dp
-  val avatarSize = (screenWidth * 0.06f).coerceIn(24.dp, 32.dp)
-  val numberOfParticipants = trip.participants.size
+fun DisplayParticipants(trip: Trip, userViewModel: UserViewModel) {
+  val numberOfParticipants = trip.participants.size - 1
```

Comment:
Why are you subtracting 1 here?

Reply:
I was having some issues with the OverviewScreen so I decided to use the one in the main branch. These changes were already there.

Reply:
I see, I don't know why they appear as your changes, but Mihai subtracted 1 for the user to not count as a participant to the trip. :)